src/targets/brew.ts

```diff
@@ -163,7 +163,8 @@ export class BrewTarget extends BaseTarget {
     const checksums: any = {};
 
     await mapLimit(filesList, MAX_DOWNLOAD_CONCURRENCY, async file => {
-      checksums[file.filename] = await this.artifactProvider.getChecksum(
+      const key = file.filename.replace(version, 'latest');
```

Comment:
To make it more visually noticeable in the resulting key, wdyt about replacing the version with "__VERSION__"?
"latest" might be a bit misleading (it's somewhat subjective, and might potentially refer to the very latest version of the package, which changes over time).

Reply:
makes sense! I'll change it